tests/unit/utils/urls.spec.js

```diff
@@ -35,6 +35,13 @@ describe('urls', () => {
         expected: `${PROTOCOL}//${HOST}/${PATHNAME}`,
         location: DEFAULT_LOCATION
       },
+      {
+        path: '',
+        websockets: false,
+        baseOnly: true,
+        expected: `${PROTOCOL}//${HOST}/`,
+        location: DEFAULT_LOCATION
+      },
```

Comment:
I pushed a commit with this test, and the docs & camel-cased parameter name. I've approved it @hjoliver . If you are OK with the changes feel free to merge it, otherwise we can drop my commit :+1:

Reply:
Thanks for doing the test!